repository_service_tuf_api/common_models.py

```diff
@@ -86,6 +86,36 @@ class TUFSigned(BaseModel):
 
     class Config:
         fields = {"type": "_type"}
+        # https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.extra
+        model_config = ConfigDict(
+            extra="allow",
+        )
+        extra = (
+            Extra.allow
+        )  # allow extra unrecognized fields (but it will be validated)
+
+    # Custom Validator for the extra fields (TUF unrecognized_fields)
+    @root_validator(pre=True)
+    def validate_unrecognized_fields(
+        cls, values: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        all_required_field_names = {
+            field.alias
+            for field in cls.__fields__.values()
+            if field.alias != "extra"  # to support alias
```

Comment:
Again, sorry for my ignorance, but where is the "extra" alias defined?

Reply:
Oops